Tests/ProjectSpecTests/SpecLoadingTests.swift

```diff
@@ -712,6 +712,7 @@ class SpecLoadingTests: XCTestCase {
                     "language": "en",
                     "region": "US",
                     "disableMainThreadChecker": true,
+                    "stopOnEveryMainThreadCheckerIssue": false,
```

Comment:
Can we set this to true, otherwise we're not testing the parsing as false is the default

Reply:
My understanding is that the default should be false because it shouldn't stop on every main thread checker issue. Am I right?

Reply:
@yonaskolb / @brentleyjones Can I please get a bit of help on this?

Reply:
@ionutivan you are correct about the default, what @yonaskolb was asking for is that we set it to `true` in a test, so we can test the parsing.

Reply:
@brentleyjones Thank you for the help!